modules/packages/TOML.chpl

```diff
@@ -1217,11 +1218,11 @@ module TomlReader {
             ld = "^\\d{4}-\\d{2}-\\d{2}",
             ti = "^\\d{2}:\\d{2}:\\d{2}(.\\d{6,})?";
 
-      const pattern = compile('|'.join(bracketContents,
-                                       brackets,
-                                       tblName,
-                                       doubleQuotes,
+
```

Comment:
I prefer avoiding double newlines. For this PR either way is fine.

Reply:
You got it boss